app/controllers/workflow_executions/submission_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module WorkflowExecutions
+  # Workflow submission controller
+  class SubmissionController < ApplicationController
```

Comment:
Can you change this controller to only respond to `turbo_stream` requests, since we don't intend this to be used otherwise.

Reply:
Updated in 06a78a914f95e2cf223be2eda103b91f2b6bb42a